vagrant/lib/forklift/box_distributor.rb

```diff
@@ -232,6 +232,7 @@ def configure_libvirt(machine, box, networks = [])
         p.memory = box.fetch('memory').to_i * @settings['scale_memory'].to_i if box.fetch('memory', false)
         p.machine_virtual_size = box.fetch('disk_size') if box.fetch('disk_size', false)
         p.management_network_domain = create_domain(box) if p.respond_to?(:management_network_domain)
+        p.qemu_use_session = @settings.fetch('libvirt_qemu_use_session', false)
```

Comment:
I actually like the session since it requires less privileges. Perhaps we should use the default of the system, but allow overriding?
```suggestion
        p.qemu_use_session = @settings['libvirt_qemu_use_session'] if @settings.key?('libvirt_qemu_use_session')
```

Reply:
Wait, how is that different logic than what I've written? If the setting doesn't exist, `fetch` will default to `false`

Reply:
Fedora 30 defaults to true and you don't cover that I think.

Reply:
What do you mean? I tested this on F30, and adding the setting in settings.yaml overrides the default of `false`. Are you seeing other behavior?

Reply:
My point is that on Fedora 30 the built in default value is `true`. My reading of the current code is that if you do nothing, it will default to `false`. I would like it to default to the built in value.

Reply:
Ah, I see. I'd like it default to false because that makes it work the way it worked before F30 (no unexpected changes for our forklift users who upgraded from F29). This is especially important if you have a bunch of existing VMs and existing libvirt networks on qemu://system. Also remember that this is all for development, so security isn't really a concern IMO (the only reason you'd want to use qemu://session, afaik).

Reply:
@ekohl convinced? ;)

Reply:
No :) My reasoning is that new users who haven't been migrated should work out of the box. Because of the user session you don't get a permission pop up and/or need to be added to the libvirt group